service/src/main/resources/quips/complibot/winsults.json

```diff
@@ -38,5 +38,45 @@
     "usageRules":[
       
     ]
+  },
+  {
+    "quipGroup":"BET_ON_YOU",
+    "text":"I wouldn't bet on you to do well in life. It's technically not gambling if it's a sure thing.",
+    "ssml":"I wouldn't bet on you to do well in life.<break /> It's technically not gambling if it's a sure thing.",
+    "usageRules":[
+      
+    ]
+  },
+  {
+    "quipGroup":"THE_LOSER",
+    "text":"I'm going to start referring to you as the loser, because you make me lose myself in your eyes every time I see you.",
+    "ssml":"I'm going to start referring to you as the loser, because you make me lose myself in your eyes every time I see you.",
+    "usageRules":[
+      
+    ]
+  },
+  {
+    "quipGroup":"JACKASSES",
+    "text":"You remind me of this one donkey I knew.  You carry with ease what a bunch of other jackasses couldn't handle.",
```

Comment:
"You remind me of a jackass. Because you're able to carry a ton of burdens with ease."